src/handlers/push_message.rs

```diff
@@ -67,7 +79,7 @@ pub async fn handler(
             "notification ({}) already received for client ({})",
             body.id, id
         );
-        return Ok(Response::new_success(StatusCode::ACCEPTED));
+        return Ok(Response::new_success(StatusCode::OK));
```

Comment:
This should be accepted - we offer no guarantees that it will be delivered

Reply:
It's to distinguish whether we actually did anything or whether it was a dupe

Reply:
Okay!